ingestion_tools/scripts/data_validation/standardized/tests/test_alignments.py

```diff
@@ -89,4 +91,19 @@ def test_tilt_tiltseries_range(
             + f"\nRange: {alignment_tiltseries_metadata['tilt_range']['min']} to {alignment_tiltseries_metadata['tilt_range']['max']}, with step {alignment_tiltseries_metadata['tilt_step']}"
         )
 
+    @allure.title("Alignment: tilt axis angle in mdoc file matches that in the alignment metadata [per_section_alignment_parameters.in_plane_rotation] (+/- 10 deg)")
+    def test_mdoc_tilt_axis_angle_in_alignment_per_section_alignment_parameters(self, mdoc_data: pd.DataFrame, alignment_tiltseries_metadata: dict[str, dict]):
+        per_section_alignment_parameters = alignment_tiltseries_metadata.get("per_section_alignment_parameters")
+        if not per_section_alignment_parameters:
+            pytest.skip("Alignment metadata missing per_section_alignment_parameters.")
+        errors = []
+        for i, psap in enumerate(per_section_alignment_parameters):
+            if (in_plane_rotation := psap.get("in_plane_rotation")) is not None:
```

Comment:
iirc, the in_plane_rotation is a 2x2 matrix. should there be a conversion for it before the following check?

Reply:
Is there a specific way we do that conversion?